modules/enabling-additional-enabled-capabilities.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_additional_enabled_capabilities_{context}"]
+= Enabling the cluster capabilities by setting additional enabled capabilities
+
+As a cluster administrator, you can enable the cluster capabilities by setting `additionalEnabledCapabilities`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the link:https://stedolan.github.io/jq/[jq package].
+
+.Procedure
+
+. View the additional enabled capabilities by running the following command:
++
+[source,terminal]
+----
+$ oc get -o json clusterversion version | jq -c '.spec.capabilities.additionalEnabledCapabilities'
+----
+
++
+.Example output
+[source,terminal]
+----
+["openshift-samples"]
+----
+
+. To set the `additionalEnabledCapabilities` run the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion/version --type merge -p '{"spec":{"capabilities":{"additionalEnabledCapabilities":["openshift-samples", "marketplace"]}}}'
+----
+
++
+[IMPORTANT]
+====
+It is not possible to disable a capability which is already enabled in a cluster. The cluster version Operator (CVO) continues to reconcile the capability which is already enabled in the cluster.
+====
+
+
+If you try to disable a capability, then CVO shows the divergent spec:
+[source,terminal]
+----
+oc get -o json clusterversion version | jq -r '.status.conditions[] | select(.type == "ImplicitlyEnabledCapabilities") | .status + " " + .reason + ": " + .message'
+----
+
+.Example output
+[source,terminal]
+----
+True CapabilitiesImplicitlyEnabled: The following capabilities could not be disabled: openshift-samples
+----
+
+[NOTE]
+====
+During cluster upgrades some new capability might get enabled, if the corresponding manifest for the capability is now part of a capability, which was enabled in the cluster before the upgrade is performed.
```

Comment:
`get enabled` -> `be enabled`. Up to you thought!

Reply:
@kelbrown20 SME is reworking on the NOTE block.